compiler/src/dotty/tools/dotc/core/Definitions.scala

```diff
@@ -932,6 +934,9 @@ class Definitions {
   @tu lazy val TupleTypeRef: TypeRef = requiredClassRef("scala.Tuple")
   def TupleClass(using Context): ClassSymbol = TupleTypeRef.symbol.asClass
     @tu lazy val Tuple_cons: Symbol = TupleClass.requiredMethod("*:")
+  @tu lazy val TupleModule: Symbol = requiredModule("scala.Tuple")
+    @tu lazy val Tuple_fromArray: Symbol = TupleModule.requiredMethod("fromArray")
```

Comment:
Seems like `Tuple_fromArray` is not used. Do we need it?

Reply:
Removed